content/concept-stand-alone-mode.md

```diff
@@ -0,0 +1,103 @@
+
+**Address:** `rHb9CJAWyB4rj91VRWn96DkukG4bwdtyTh`
+
+**Secret:** `snoPBrXtMeMyMHUVTgbuqAfg1SUTb` ("masterpassphrase")
+
+**Caution:** If you create a new genesis ledger, the hard-coded default [Reserve](concept-reserves.html) is **200 XRP** minimum for funding a new address, with an increment of **50 XRP** per object in the ledger. These values are higher than the current reserve requirements of the production network. (See also: [Fee Voting](concept-fee-voting.html))
+
+
+
+Load Saved Ledger
+-------------------------------------------------------------------------------
+You can start with a ledger version that was saved to disk if your `rippled` server was previously synced with the Ripple peer-to-peer network (either the production network or the [Test Net](tutorial-rippled-setup.html#parallel-networks)).
+
+### 1. Start `rippled` normally. ###
+
+To load an existing ledger, you must first retrieve that ledger from the network. Start `rippled` in networked mode as normal:
+
+```
+rippled --conf=/path/to/rippled.cfg
+```
+
+### 2. Wait until `rippled` is synced. ###
+
+Use the [`server_info` command](reference-rippled.html#server-info) to check the state of your server relative to the network. Your server is synced when the `server_state` value shows any of the following values:
+
+* `full`
+* `proposing`
+* `validating`.
+
+### 3. (Optional) Retrieve specific ledger versions. ###
+
+If you only want the most recent ledger, you can skip this step.
+
+If you want to load a specific historical ledger version, use the [`ledger_request` command](reference-rippled.html#ledger-request) to make `rippled` fetch it. If `rippled` does not already have the ledger version, you may have to run the `ledger_request` command multiple times until it has finished retrieving the ledger.
+
+If you want to replay a specific historical ledger version, you must fetch both the ledger version to replay and the ledger version immediately before it.
+
+### 4. Shut down `rippled`. ###
+
+Use the [`stop` command](reference-rippled.html#stop):
+
+```
+rippled stop --conf=/path/to/rippled.cfg
+```
+
+### 5. Start `rippled` in stand-alone mode. ###
+
+To load the most recent ledger version, you can use the `-a` and `--load` options when starting the server:
+
+```
+rippled -a --load --conf=/path/to/rippled.cfg
+```
+
+To instead load a specific historical ledger, use the `--load` parameter along with the `--ledger` parameter, providing the ledger index or identifying hash of the ledger version to load:
+
+```
+rippled -a --load --ledger 19860944 --conf=/path/to/rippled.cfg
+```
+
+### 6. Manually advance the ledger. ###
+
+When you load a ledger with `--ledger` in stand-alone mode, it goes to the current open ledger, so you must [manually advance the ledger](#advancing-ledgers-in-stand-alone-mode):
+
+```
+rippled ledger_accept --conf=/path/to/rippled.cfg
+```
+
+
+Advancing Ledgers in Stand-Alone Mode
```

Comment:
It's a little weird to see them side-by-side like that, I agree, but they're separate for good reason. Step 6 is a step of the process with an example. The "Advancing Ledgers in Stand-Alone Mode" header is meant to be linked from other docs (e.g. the multi-signing tutorial)